### ISSUE TITLE
Expose a service for applying mask in typescript

### ISSUE THREAD

**Issue by kevinrstone** [JsDaddy/ngx-mask#323]
Please advise how to apply a mask in typescript code. 

Correct me if I am wrong, but I do not see any exposed service that I can use to do this. MaskService cannot be injected into a service, because it requires an ElementRef, which throws an error on injection `NullInjectorError: No provider for ElementRef!`.

There is another service MaskApplierService, but it is not exposed in the public_api. In any case there is no documentation on the difference between these two services.

If it is not currently possible to apply the mask outside of an Angular template, please add as a feature the correct service to use, or a utility class that can be imported. The use case is for data export.

**Comment by ValeriaKochegarova** [comment]
@kevinrstone  could you provide an example ?

**Comment by kevinrstone** [comment]
How do I do something like this? It is not currently possible to import MaskService. Each item in our database may have a different mask, depending on region. It is not desirable to attempt to reproduce the code for applying and displaying a mask.

```@Injectable()
export class ExportDataService {
    constructor(
        private maskService: MaskService
    ) {}

    exportData(item: any) {
        
        let phoneNumber: string = maskService.applyMask(item.rawPhoneNumber, item.maskString);
        // export phone number to CSV, Email, other service, etc
    }
}
```

**Comment by douggass** [comment]
solution? I have same problem.

**Comment by ValeriaKochegarova** [comment]
@kevinrstone @douggass we fixed this , you can use in mask 7.9.8 version 
For using you should use MaskApplierService 
Example: 
``` 
constructor(private maskService: MaskApplierService) {} 
  exportData() {
    const phoneNumber: string = this.maskService.applyMask('12345678901', '000-0000-00-00');
}
```

**Comment by MRMokwa** [comment]
Why is nfx-mask.d.ts like this?
`export { MaskApplierService as ɵa } from './app/ngx-mask/mask-applier.service';`

I'm on 7.9.10.
